csvw/csvw.go

```diff
@@ -191,8 +191,8 @@ func (csvw *CSVW) AddNotes(metadata *dataset.Metadata, url string) {
 		}
 	}
 
-	if metadata.UsageNotes != nil {
-		for _, u := range *metadata.UsageNotes {
+	if metadata.DatasetDetails.UsageNotes != nil {
```

Comment:
Only question I have is can `DatasetDetails` be nil? If so we should probably add a defensive check for that too. Other than that happy to approve.